clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -25,28 +25,51 @@ import shapeless.CNil
   */
 sealed trait ClioCommand
 
+@CommandName(ClioCommand.getServerHealthName)
+case object GetServerHealth extends ClioCommand
+
+@CommandName(ClioCommand.getServerVersionName)
+case object GetServerVersion extends ClioCommand
+
+@CommandName(ClioCommand.getWgsUbamSchemaName)
 case object GetWgsUbamSchema extends ClioCommand
 
+@CommandName(ClioCommand.addWgsUbamName)
 final case class AddWgsUbam(metadataLocation: String,
                             @Recurse transferWgsUbamV1Key: TransferWgsUbamV1Key)
     extends ClioCommand
 
+@CommandName(ClioCommand.queryWgsUbamName)
 final case class QueryWgsUbam(
   @Recurse transferWgsUbamV1QueryInput: TransferWgsUbamV1QueryInput,
+  includeDeleted: Boolean = false
 ) extends ClioCommand
 
+@CommandName(ClioCommand.moveWgsUbamName)
 final case class MoveWgsUbam(
   @Recurse metadata: TransferWgsUbamV1Metadata,
   @Recurse transferWgsUbamV1Key: TransferWgsUbamV1Key
 ) extends ClioCommand
 
+@CommandName(ClioCommand.deleteWgsUbamName)
 final case class DeleteWgsUbam(
   @Recurse metadata: TransferWgsUbamV1Metadata,
   @Recurse transferWgsUbamV1Key: TransferWgsUbamV1Key
 ) extends ClioCommand
 
 object ClioCommand extends ClioParsers {
 
+  // Names for generic commands.
+  val getServerHealthName = "get-server-health"
+  val getServerVersionName = "get-server-version"
+
+  // Names for WGS uBAM commands.
+  val getWgsUbamSchemaName = "get-wgs-ubam-schema"
```

Comment:
As in my comments on @ahakone 's checkin, I think we should consider putting the different bit of the name at the start or the end of the string instead of the middle. E.g. `wgs-ubam-get-schema` and `wgs-ubam-add`. Or maybe what we want is `add wgs-ubam` E.g. the command line args are `<operation>` `<type>`. Am just thinking ahead to the point where we have 5-6 indexes instead of just two.

Reply:
I'll shuffle it around to match her PR. Agreed `<operation>` `<type>` would be ideal and, we should try moving to it between the gvcf stuff and whatever comes next! But I wouldn't be surprised if it takes another epic adventure with shapeless to make it work.